src/main/kotlin/no/nav/familie/ef/sak/beregning/barnetilsyndto/BeregningBarnetilsynRequest.kt

```diff
@@ -0,0 +1,36 @@
+package no.nav.familie.ef.sak.beregning
+
+import java.math.BigDecimal
+import java.time.YearMonth
+import java.util.UUID
+
+data class BeregningBarnetilsynRequest(
+        val utgiftsperioder: List<UtgiftsperiodeDto>,
+        val kontantstøtteperioder: List<KontantstøttePeriodeDto>,
+        val tilleggsstønadsperioder: List<TilleggsstønadPeriodeDto>
+)
+
+data class TilleggsstønadPeriodeDto(
```

Comment:
Denne og KontantstøttePeriodeDto er like, hvis dere bruker den samme klassen, så går det å gjennbruke metoder for begge disse

Reply:
Då får man slått ihop `finnTillegstønadBeløp` og `finnKontantstøtteBeløp`

Reply:
Vi testet. Ja det var fint å få slettet tre linjer kode (finnTillegstønadBeløp), men vi mistet også noe på veien. 

Vanskelig å gi den nye dto'en et godt navn (InntektsperiodeDto ???) 
Ble litt vanskeligere å lese kode noen steder (ikke veldig, men nok til at jeg vipper i retning av å beholde det som det er nå)

Reply:
`PeriodeMedBeløp` er brukt i vedtak

Reply:
Bruker også PeriodeMedBeløp i frontend :)